app/views/reviewRegistration.html

```diff
@@ -415,13 +415,15 @@ <h2 class="page-title border" translate>Payment</h2>
       <p ng-if="!allRegistrantsValid()" style="color:red;" translate>
         Please fill in all required fields in red before submitting.
       </p>
-      <input
-        type="button"
+      <button
         class="btn btn-lg btn-success btn-important confirm-registration"
         ng-click="confirmRegistration()"
-        value="Confirm"
         ng-disabled="registerDisabled()"
-      />
+      >
+        <translate ng-if="!submittingRegistration">Confirm</translate>
+        <translate ng-if="submittingRegistration">Submitting..</translate>
```

Comment:
```suggestion
        <translate ng-if="submittingRegistration">Submitting...</translate>
```
Are ellipses usually 3 dots? Idk 🤷

Reply:
Went ahead and added the extra dot, just rebased and squashed the commit 👍